src/settings.ts

```diff
@@ -71,4 +71,9 @@ export default interface IAuth0Settings {
    * Settings for the OIDC Client which performs the code exchange.
    */
   oidcClient?: OidcClientSettings;
+
+  /*
+   * Function to format and retrieve the user object to be stored in the cookie
+   */
+  formatUserObject: (user: any) => any;
```

Comment:
I'm not a huge fan of this name, I'm going to mull it over for a bit.  I think we need something more 'action'y.

Reply:
Maybe we take a page from Passport's book and call it 'serializeUser'?  Thoughts?

Reply:
That sound good!